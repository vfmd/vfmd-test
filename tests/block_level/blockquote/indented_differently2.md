> Not indented

 > Indented by one space

> Not indented

  > Indented by two spaces

> Not indented

   > Indented by three spaces
