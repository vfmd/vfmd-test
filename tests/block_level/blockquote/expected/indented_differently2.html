<blockquote>
  <p>Not indented</p>

<p>Indented by one space</p>

<p>Not indented</p>

<p>Indented by two spaces</p>

<p>Not indented</p>

<p>Indented by three spaces</p>
</blockquote>
